src/app/views/settings/general/general.controller.coffee

```diff
@@ -48,7 +50,7 @@
         vm.settingsModel = response.tenant.data.frontend_config
         vm.originalSettings = angular.copy(vm.settingsModel)
         vm.settingsSchema = response.tenant.data.config_schema
-        validateAppList(_.map(response.marketplace.data.apps, 'nid'))
+        validateAppList(_.map(response.marketplace.data.apps, 'nid'), _.map(_.filter(response.marketplace.data.products, (product) -> product.local), 'nid'))
```

Comment:
You can use the `_.property` iteratee shorthand:
`_.filter(esponse.marketplace.data.products, 'local')`